### ISSUE TITLE
Mocha tests do not continue after [ENTER] with browser.debug()

### ISSUE THREAD

**Issue by tizmagik** [xolvio/chimp#498]
##### Expected behaviour

Enter key continues Mocha test after `browser.debug()`
##### Actual behaviour

Enter key does not continue tests
##### Steps to reproduce the problem

Create a test like so:

``` js
describe('Mocha test', () => {
  before(() => {
    browser.url('/');
  });
  it('should run', () => {
    browser.debug();
    expect(true);
  });
});
```
##### Used version
- Chimp: 0.41.1
- Node.js: 6.3.1
- Operation system: macOS Sierra

It works when using the Jasmine test runner.

---

_Join our Slack [xolv.io/community](http://xolv.io/community) #chimp channel, where you can find help and help others._

**Comment by samhatoum** [comment]
Just confirmed this bug

**Comment by kyleian** [comment]
Confirmed same issue with Cuke on Chimp 0.41.2 / NodeJS 5.8.

**Comment by samhatoum** [comment]
It looks like the stdin is not being piped through to the underlying process. Working on a fix...

**Comment by lucetius** [comment]
Fixed in #565
